usr/share/rear/verify/COMMVAULT/default/390_create_ramdisk.sh

```diff
@@ -0,0 +1,30 @@
+#
+# Commvault checks for free disk space in /opt /var and /tmp
+#
+# Our initramfs doesn't report any free disk space, so that we have to setup
+# a tmpfs for that. 1500MB ist just a guess, if your system blows because of this
+# you should feel free to determine a better way or a smaller ramdisk size to suffice
+# (be warned, lots of tests are required for this ...)
+#
+
+# we want to do this only once ...
+if ! test -d /ramdisk ; then
```

Comment:
Only a coding style suggestion:
Personally I prefer "return early, return often" style, see
https://github.com/rear/rear/wiki/Coding-Style#return-early-return-often

So in this case it would be
```
...
test -d /ramdisk && return

mkdir /ramdisk
...
```
without all actual code nested in an if clause.

Reply:
agree